### PR TITLE
Add "Congratulations!" dialog on colony creation

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
- To test the dialog, just create a colony. It should appear when you are redirected to the colony dashboard.

FireShot Capture 206 - Extensions - Colony - - localhost

### CODE REVIEW

File: src/components/common/InvitationBlock/InvitationBlock.tsx [hunk 1]
```diff
@@ -23,30 +24,43 @@ const MSG = defineMessages({
...
-const InvitationBlock = () => {
+interface Props {
+  showDescription?: boolean;
+}
+
+const InvitationBlock = ({ showDescription = true }: Props) => {
   const { user } = useAppContext();
   const invitationCode = user?.privateBetaInviteCode?.id;
-  const inviteLink = `app.colony.io/createcolony/${invitationCode}`;
+  const inviteLink = `app.colony.io/create-colony/${invitationCode}`;
```
Comment (rdig) [inline src/components/common/InvitationBlock/InvitationBlock.tsx:43]: Why is the url hardcoded ? And why is this not a hook that just generates your invite code ?

File: src/routes/ColonyRoute.tsx [hunk 2]
```diff
@@ -15,11 +18,13 @@ const ColonyRoute = () => (
       <ColonyDecisionProvider>
         <UserTokenBalanceProvider>
           <MemberModalProvider>
-            <UserTransactionContextProvider>
-              <ColonyLayout>
-                <Outlet />
-              </ColonyLayout>
-            </UserTransactionContextProvider>
+            <ColonyCreatedModalProvider>
```
Comment (rdig) [inline src/routes/ColonyRoute.tsx:21]: I feel this will get out of hand fast if we end up having a provider for every modal out there.

**Review by rdig — DISMISSED** [review]
Besides the hardcoded url and the invite link not being a hook, I'm all good here.

However is up to @melyndav to properly check the styles

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @ArmandoGraterol! Great work on this, and nice to see this feature come to live.

Can I request a couple of minor UI changes to both desktop and mobile variations. 

1. The padding between the top icon and the X close icon should smaller as per the Figma design just to reduce the model height.
image

2. Can we change the text size on the following text in the second section 'You can invite 1 member... ' to be the following:

image

2. Can we reduce the space between these two?

image

3. On mobile, the model should be a pop-up with an overlay, not a full screen model -

https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=11978%3A126485&mode=dev&t=brqRb02uKRH5EEbq-1

4. There invite member box should be the mobile version. There's issues with the button. 

5. The little party hat icon should be #101828 (text-gray-900) it's currently #000000.

File: src/routes/ColonyRoute.tsx (hunk 2 above)
Comment (ArmandoGraterol) [reply src/routes/ColonyRoute.tsx:21]: Agreed

File: src/components/common/InvitationBlock/InvitationBlock.tsx (hunk 1 above)
Comment (ArmandoGraterol) [reply src/components/common/InvitationBlock/InvitationBlock.tsx]: Added a new hook, let me know if something like this is what you had in mind 👍

**Comment by ArmandoGraterol** [comment]
@melyndav I think all the changes you requested have been addressed! Let me know if there's something else or if I got something wrong :P

**Comment by chmanie** [comment]
I rebased this on `master`

**Review by melyndav — APPROVED** [review]
Awesome work @ArmandoGraterol! Thanks for all your updates. I appreciate it. :)

**Pull request merged**